libbz2-rs-sys/src/compress.rs

```diff
@@ -356,31 +356,16 @@ fn send_mtf_values(s: &mut EState) {
                 let mut cost01: u32 = 0;
                 let mut cost23: u32 = 0;
                 let mut cost45: u32 = 0;
-                let mut icv: u16;
 
-                macro_rules! BZ_ITER {
-                    ($nn:expr) => {
-                        icv = mtfv[(gs + $nn) as usize];
-                        cost01 = cost01.wrapping_add(s.len_pack[icv as usize][0]);
-                        cost23 = cost23.wrapping_add(s.len_pack[icv as usize][1]);
-                        cost45 = cost45.wrapping_add(s.len_pack[icv as usize][2]);
-                    };
+                for chunk in mtfv[gs as usize..][..50].chunks_exact(5) {
+                    for icv in chunk {
```

Comment:
Why chunks of 5?

Reply:
well, we have 50 elements, and to use `chunks_exact` and not drop any elements, your chunk size must evenly divide 50. Using `chunks` (so, non-exact) leads to worse performance. Picking either 5 or 10 does not seem to matter much.